app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,46 @@
+class InventoriesController < ApplicationController
+  def index
+    @inventories = Inventory.all
+  end
```

Comment:
- [ ] Well done on implementing the inventories list! :clap: However, it would be beneficial to only display a list of inventories created by the logged-in user."

Reply:
@LionRouge1 thank you for this. I have fixed the problem.